apartments/forms.py

```diff
@@ -32,3 +32,46 @@ class Meta:
             'long_term',
             'immediate_entry'
         ]
+
+
+class ApartmentCreationForm(forms.ModelForm):
+
```

Comment:
1) For all the validations, please see: https://stackoverflow.com/questions/12384460/allow-only-positive-decimal-numbers
2) You are specifically looking for <0, what about 0 itself?

Reply:
Ok, ill look into it and push the changes.
About 0 itself I thought that having negative rent is really wierd - you can argue that paying no rent at all is wierd too, but paying 1 dollar/shekel is also wierd - so I thought that because there's alot of wierd values here it's better to limit only the negative numbers, if you want to rent your house for free you can!